nservicebus/messaging/message-correlation.md

```diff
@@ -7,14 +7,18 @@ related:
  - nservicebus/messaging/headers
 ---
 
-Message correlation is the act of connecting instances of request messages with instances of response messages. Since all messages have a `Message Id` the `Correlation Id` is just a reference back to the specific message instance that caused this message to be sent back as a reply.
+Message correlation connects instances of request messages with instances of response messages. Each message has a `Message Id` and the `Correlation Id` is a reference back to the specific message instance that caused this message to be sent back as a reply.
```

Comment:
What's incorrect about the statement in the headers doco?

The differences between `NServiceBus.RelatedTo` and `NServiceBus.CorrelationId` do exist. One of them is that the correlation ID is customizable by the user. Do you think it would be good to expand on this as well?